packages/wonder-blocks-icon-button/src/components/activity-icon-button.tsx

```diff
@@ -46,6 +46,11 @@ type Props = BaseIconButtonProps &
          * - `neutral` is used for buttons that indicate a neutral action.
          */
         actionType?: ActivityIconButtonActionType;
+
+        /**
+         * A Phosphor icon asset (imported as a static SVG file), or an element.
+         */
+        icon: PhosphorIconAsset | React.ReactElement;
```

Comment:
Alternatively, instead of `ReactElement`, we might be able to enforce us only passing our own wonderblocks `<Icon />` type. LMK what y'all think.